### DISCUSSION TITLE
Move from Quarantine to Inbox

### DISCUSSION THREAD

**Discussion by Fonsos** [simple-login/app#1320] [Feature Requests]
So I got some emails into quarantine, but they weren't spam and I would like to get them in my inbox.

How about to add a button un un-quarantine emails and send them into our inbox? If necessary I think log-in again for safety may be a security option we could have.

Thanks for your awesome work!

**Comment by nguyenkims** [comment]
@Fonsos Emails are usually put into quarantine as they fail the Spam check and if they are forwarded to your mailbox, your mailbox service can bounce it back. Currently a workaround is to download the email as an .eml file and if it's an important email, import it back to your mailbox so it can be indexed. We are still looking for a solution for this so any idea is welcome!

**Reply by rowatt** [reply]
@nguyenkims The workaround is OK for an occasional mail which gets trapped in quarantine, however it is proving to be very inconvenient when trying to have an email conversation. I am currently liaising with a company for an ongoing tech support issue. Their mails to me are I assume sent by their support system, and every single one goes to quarantine. I told them about this, but they have been unable to resolve so far.

When I receive an email from them which I want to reply to, I need to download the .eml file, open it in Apple mail, forward it to my Google account, look up the reverse alias address and paste that in.

Would it be possible to add a switch on the settings for any alias which, if set, bypasses all security checks for that alias? Or at least the DMARC/DKIM/SPF ones which seem to trigger most of the problematic but legitimate emails I receive?

That would keep the initial warning, but would allow people to override it for aliases where the sender isn't able (or willing) to fix their outbound email setup.

A global switch to turn off all these checks would also work well for me - hardly any of the quarantine or phishing warnings added by SimpleLogin are bad emails. Almost all are legitimate, but misconfigured.

**Reply by nguyenkims** [reply]
Thanks for the info. Can you please send me the alias and the timestamp when this happened so I can investigate? My email is son at simplelogin.io

**Reply by huntfx** [reply]
I support the idea of an option to disable all security checks for an alias.

I have a "work > personal" forward email alias, and it turns out someone making a suggestion on Google Docs will trigger the quarantine.

**Comment by x80486** [comment]
There should be a button to just forward the e-mail to the inbox, regardless. Many systems won't let you import `.eml` files.

I just got one that it's indeed not spam.

SimpleLogin shouldn't need to flag messages because it's a forwarding system.
